src/lib.rs

```diff
@@ -140,3 +142,9 @@ fn emit_diagnostics<'a>(
 
     Ok(())
 }
+
+/// Returns a vectory containing a single source for the current working
+/// directory.
+fn cwd_source() -> Vec<Source> {
+    vec![Source::Directory(PathBuf::from("."))]
```

Comment:
Porting my suggestion from https://github.com/stjude-rust-labs/wdl/pull/476/files#r2140983731
```suggestion
    vec![Source::Directory(PathBuf::from(std::path::Component::CurDir.as_os_str()))]
```